app/models/ems_refresh/parsers/scvmm.rb

```diff
@@ -434,6 +434,8 @@ def process_iso_image(vm)
     end
 
     def process_vm_storages(properties)
+      return if properties[:VirtualHardDisks].nil?
```

Comment:
@chessbyte 
See the method *link_habtm(...)* in the file:
https://github.com/bronaghs/manageiq/blob/scvmm_no_disks_error/app/models/ems_refresh/link_inventory.rb#L110

The VM storages hash is passed into this method and is checked for nil. An empty array is supported too.